### PR TITLE
Add CompanyServiceImpl Test

### PR DESCRIPTION
**Pull request by NyorJa**
## PR Checklist

- [x] Read the [contribution guidelines](/CONTRIBUTING.md)
- [x] Added a brief description of these changes to the top of the [changelog](/CHANGELOG.md)
- [x] Changes in code meet test criteria (running `mvn test` returns exit code 0, without errors)

## PR Type

- [x] Test-related

## Summary

Validate the proper mapping behaviour of CompanyServiceImpl method "readDetails"

### CODE REVIEW

**Comment by NyorJa** [comment]
Hi @bglamadrid,

Please check my changes in Changelog.md if its correct format

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Tests
+- CompanyServiceImpl - validate the proper mapping behaviour of method "readDetails"
+
```
Comment (bglamadrid) [inline CHANGELOG.md:11]: There is a `Tests` section at line 34, you can move this change there

**Review by bglamadrid — APPROVED** [review]
Outstanding work @NyorJa 💯

I do have one minor concern, a single comment below that you may address if possible. Besides that, this is exactly what was required 👌

I will merge this PR no later than 6PM UTC.
Thank you once again!

File: CHANGELOG.md (hunk 1 above)
Comment (NyorJa) [reply CHANGELOG.md:11]: oops..ill fix that.

**Pull request merged**